docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -309,3 +347,8 @@ ______________
     :language: python
     :lines: 34-64
     :tab-width: 0
+
+NLP Example
+___________
+
+Currently unavailable, will be added in future releases.
```

Comment:
Why add it, though? Shouldn't we remove this and add an issue? (I guess unavailable because you want to show token classification?)

Reply:
For sake of clarity I think its better with this section for users to know it will be added in the future, but dont have strong feeling either way